visivo/models/sources/snowflake_source.py

```diff
@@ -41,35 +41,38 @@ class SnowflakeSource(Source):
         None,
         description="The access role that you want to use when running queries.",
     )
+    timezone: Optional[str] = Field(
+        None,
+        description="The timezone that you want to use by default when running queries.",
+    )
 
     type: SnowflakeType
+    connection_pool_size: Optional[int] = Field(
```

Comment:
I thought this would be inherited from the parent class now.

Reply:
I think we're still setting them on each individual source.